subprojects/docs/src/snippets/tutorial/configureObject/kotlin/build.gradle.kts

```diff
@@ -1,12 +1,12 @@
-import java.text.FieldPosition
+class UserInfo(var name: String? = null, var email: String? = null)
```

Comment:
I had to use a custom type instead of a JDK one or else I would get:

```
* What went wrong:
  Configuration cache state could not be cached: field 'actions' from type 'org.gradle.api.DefaultTask': error writing value of type 'java.util.ArrayList'
  > Configuration cache state could not be cached: field 'closure' from type 'org.gradle.api.internal.AbstractTask$ClosureTaskAction': error writing value of type 'build_417ufrclhd0hxg7pfdp4wglsj$_run_closure1$_closure2'
     > Configuration cache state could not be cached: field 'pos' from type 'build_417ufrclhd0hxg7pfdp4wglsj$_run_closure1$_closure2': error writing value of type 'groovy.lang.Reference'
        > Configuration cache state could not be cached: field 'value' from type 'groovy.lang.Reference': error writing value of type 'java.text.FieldPosition'
           > Unable to make field private java.text.Format$Field java.text.FieldPosition.attribute accessible: module java.base does not "opens java.text" to unnamed module @9203747
```